test/helloworld.eventuals.cc

```diff
@@ -40,7 +40,7 @@ Task::Of<void>::Raises<std::exception> Greeter::TypeErasedService::Serve() {
                      return UnaryPrologue(call)
                          | Then(Let([&](auto& request) {
                               return Then(
-                                  [this,
+                                  [&,
```

Comment:
Capturing `this` is explicitly _less_ so I'd prefer it! It's not clear why you needed to change this?